### ISSUE TITLE
The enhancement bonus for Magic Weapons does not apply to damage

### ISSUE THREAD

**Issue by freohr** [vttred/ose#181]
The bonus correctly applies to the Attack Roll, but not to the Damage Roll.
[Relevant SRD link](https://oldschoolessentials.necroticgnome.com/srd/index.php/Weapons#Enchanted_Swords_and_Weapons)

![fvtt_ose_magic_bonus_not_applied_damage](https://user-images.githubusercontent.com/3462951/168869843-9bf3c627-2471-4f49-bf25-cf9d298a369a.png)

**Comment by justinthejaguar** [comment]
I’m pretty sure this is so you can decide to have to hit bonuses independent of damage bonuses. It has been useful for home brewed items. To increase the damage you can add +1 to the die roll.

**Comment by anthonyronda** [comment]
Yes, this has been discussed before, not a bug necessarily but it is an example where the system diverges from rules as written. It would be nice to make this less confusing for users somehow, but elegant ways to do so escape me at the moment. Perhaps two checkboxes under bonus that enable when a non-zero number is placed there? "Hit" and "Damage?"

**Comment by Godforsaken84** [comment]
![Screenshot 2022-05-25 055819](https://user-images.githubusercontent.com/100036544/170171353-cccd36e2-ba33-437c-afb0-2f6317915e85.png)
I think that adding a checkbox like the "slow weapon" checkbox just below the bonus box should be intuitive enough.

**Comment by anthonyronda** [comment]
@freohr what do you think? I personally love the use of the icons here

**Comment by Godforsaken84** [comment]
It should be on. It is more common for weapons to have a regular enchantment bonus to attack and damage.

**Comment by anthonyronda** [comment]
@justinthejaguar @Godforsaken84 AYYY lmao I was looking for this discussion in Discord for like an hour because I could've sworn it was in there 😭 so yes this came up again in reference to #410 and we all had another discussion about it 😆 

The reason why I decided against placing the checkbox in the item sheet is that I'm currently unhappy with the design of the item sheet. Since I haven't been able to start that conversation, I placed this checkbox in the Tweaks menu of the actor instead. I don't necessarily think that's a permanent home for it (I think everything in the Tweaks menu will need to find a permanent home elsewhere eventually).

That said, I think it's actually good to have it on the actor instead of the item. Math and numbers add more cognitive load than other types of input boxes. If the same player has two different items that calculate differently, I can see that increasing confusion. It's just as well that they have to input all of their damage boxes and bonus boxes exactly the same way.
